docs/products/kafka/kafka-connect/howto/jdbc-catalog-postgres.md

```diff
@@ -0,0 +1,139 @@
+  PostgreSQL instance that is publicly accessible
+- Amazon S3 bucket for storing Iceberg table data
+- IAM user or role with permissions to read and write to the S3 bucket
+- AWS access key ID and secret access key for the IAM credentials
+
+
+## Create an Iceberg sink connector configuration
+
+To configure the Iceberg sink connector with the JDBC catalog, define a JSON file that
+uses PostgreSQL for metadata and Amazon S3 for storage.
+
+:::note
+Loading worker properties is not supported. Use `iceberg.kafka.*` properties instead.
+:::
+
+```json
+{
+"name": "<connector-name>",
+"connector.class": "org.apache.iceberg.connect.IcebergSinkConnector",
+"tasks.max": "2",
+"topics": "<kafka-topics>",
+"key.converter": "org.apache.kafka.connect.json.JsonConverter",
+"value.converter": "org.apache.kafka.connect.json.JsonConverter",
+"key.converter.schemas.enable": "false",
+"value.converter.schemas.enable": "false",
+"consumer.override.auto.offset.reset": "earliest",
+"iceberg.kafka.auto.offset.reset": "earliest",
+"iceberg.tables": "<database>.<table>",
+"iceberg.tables.auto-create-enabled": "true",
+"iceberg.control.commit.interval-ms": "1000",
+"iceberg.control.commit.timeout-ms": "20000",
+"iceberg.catalog.type": "jdbc",
+"iceberg.catalog.uri": "jdbc:postgresql://<host>:<port>/<database>?user=<username>&password=<password>&ssl=require",
+"iceberg.catalog.warehouse": "s3://<bucket-name>",
+"iceberg.catalog.client.region": "<region>",
+"iceberg.catalog.io-impl": "org.apache.iceberg.aws.s3.S3FileIO",
+"iceberg.catalog.jdbc.useSSL": "true",
+"iceberg.catalog.jdbc.verifyServerCertificate": "true",
+"iceberg.catalog.s3.access-key-id": "<aws-access-key-id>",
+"iceberg.catalog.s3.secret-access-key": "<aws-secret-access-key>",
+"iceberg.catalog.s3.path-style-access": "true",
+"iceberg.kafka.bootstrap.servers": "<kafka-host>:<port>",
+"iceberg.kafka.security.protocol": "SSL",
+"iceberg.kafka.ssl.keystore.location": "/run/aiven/keys/public.keystore.p12",
+"iceberg.kafka.ssl.keystore.password": "password",
+"iceberg.kafka.ssl.keystore.type": "PKCS12",
+"iceberg.kafka.ssl.truststore.location": "/run/aiven/keys/public.truststore.jks",
+"iceberg.kafka.ssl.truststore.password": "password",
+"iceberg.kafka.ssl.key.password": "password"
+}
+```
+
+Parameters
+
+- `name`: Name of the connector.
+- `connector.class`: Connector implementation class.
+- `topics`: Apache Kafka topics with source data.
+- `iceberg.catalog.type`: Catalog type. Use `jdbc` for PostgreSQL.
+- `iceberg.catalog.uri`: JDBC connection string for PostgreSQL.
+- `iceberg.catalog.warehouse`: S3 bucket for table storage.
+- `iceberg.catalog.client.region`: AWS region where the S3 bucket is located. Required if
+  no region is set in the environment or system properties.
+- `iceberg.tables`: Target Iceberg table in `<database>.<table>` format.
+- `iceberg.tables.auto-create-enabled`: Automatically create tables if they do not exist.
```

Comment:
```suggestion
- `iceberg.tables.auto-create-enabled`: Automatically creates tables if they do not exist.
```

Reply:
Since this is a description of a setting (not a full sentence with a subject), I think the fragment "Automatically create tables if they do not exist" is more consistent with the rest of the list, which also uses imperative-style fragments